protocols/48648f/README.md

```diff
@@ -0,0 +1,32 @@
+# Nucleic acid purification from 90 patient samples
+
+### Author
+[Opentrons](https://opentrons.com/)
+
+## Categories
+* Nucleic acid purification
```

Comment:
There should be a main category and subcategory here. The main category could be `Nucleic Acid Extraction & Purification` and the subcategory could be `Nucleic Acid Purification`

Happy to give more details if needed.